plugins/CoreHome/lang/en.json

```
@@ -104,7 +104,9 @@
         "RowLimit": "Row limit",
         "CustomLimit": "Custom limit",
         "ExportFormat": "Export format",
-        "ExportTooltip": "Note: The generated export URL will only work in the current browser session. If you want to use it somewhere else you need to use an app specific token. You can configure these tokens in Admin -> Security -> Token Auths.",
+        "ExportTooltip": "Note: The generated export URL will only work in the current browser session. If you want to use it somewhere else you need to use an app specific token. You can configure these tokens in [Admin -> Security -> Token Auths]. Replace ENTER_YOUR_TOKEN_AUTH_HERE in the Export URL by your Auth token. Warning: never share the URL with the real token with anyone else.",
+        "ExportTooltipWithLink": "Note: To use the generated export URL, you will need to specify an app token auth. You can configure these tokens in %1$s[Admin -> Security -> Auths Tokens]%2$s. Replace ENTER_YOUR_TOKEN_AUTH_HERE in the Export URL by your Auth token. Warning: never share the URL with the real token with anyone else.",
```

```suggestion
        "ExportTooltipWithLink": "Note: To use the generated export URL, you will need to specify an app token auth. You can configure these tokens in %1$s[Admin -> Security -> Auths Tokens]%2$s. Replace %3$s in the Export URL by your Auth token. Warning: Never share the URL with the real token with anyone else.",
```

We shouldn't have `ENTER_YOUR_TOKEN_AUTH_HERE` in this string as well, but instead should use a placeholder to inject it. Otherwise it could happen that it is translated in different ways.